scripts/translations.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs');
+const path = require('path');
+
+const shell = require('shelljs');
+
+const main = async () => {
+  const {stdout} = shell.exec('git diff --name-only --cached', {silent: true});
+  const changes = stdout.trim().split('\n');
+
+  let packages;
+
+  try {
+    packages = fs.readdirSync(path.resolve(process.cwd(), 'packages'), 'utf8');
+
+    packages = packages.filter((pkg) => {
```

Comment:
Can't we use `startsWith` to make it more readable?

```javascript
packages = packages.filter(pkg => pkg.startsWith('strapi'));
```

Or `includes` if isn't a requirement that begins with `strapi`.

Reply:
Same job but yes we can